src/config/statsd.ts

```diff
@@ -10,6 +11,7 @@ export const globalTags = {
 	region: process.env.MICROS_AWS_REGION || "localhost"
 };
 
+const RESPONSE_TIME_HISTOGRAM_BUCKETS =	"100_1000_2000_3000_5000_10000_30000_60000";
```

Comment:
For http requests we have much smaller buckets than for overall webhooks processing because we expect our rest api reply fast. If they reply slower then few seconds, in frontend, it is considered to be a failure from the user experience point of view.

Reply:
Now that we have multiple buckets for histogram metrics do you think we should move the webhook buckets into this file?

Reply:
No, for webhook buckets we have their own. It make sense that they are different, because webhooks processing time is expected to be higher (and what normal time for webhook processing can be too slow for the frontend API)

Reply:
I wasn't saying to make them the same, I was asking if the buckets for both should live in this file so they are in a central location.

Reply:
I think it is fine if they live where they needed. Better group code and definitions by its purpose rather than by their data type.   (For example here are http metrics, and there are webhook metrics)